heron/tools/cli/src/python/execute.py

```diff
@@ -59,7 +59,7 @@ def heron_class(class_name, lib_jars, extra_jars=None, args=None, java_defines=N
   all_args += [class_name] + list(args)
 
   # set heron_config environment variable
-  heron_env = os.environ.copy()
+  heron_env = {}
```

Comment:
At first I thought we don't need to copy all the env variables, but now I think we do need all env vars. For example, Java program could fetch environmental variables.